lib/instagram_client.rb

```diff
@@ -1,8 +1,21 @@
 class InstagramClient
   INSTAGRAM_SEARCH_USERS_URI = "https://api.instagram.com/v1/users/search?client_id=#{ENV["INSTAGRAM_CLIENT_ID"]}&"
+  INSTAGRAM_FETCH_GRAMS_URI = "https://api.instagram.com/v1/users/"
+  SEED_COUNT = 5
```

Comment:
We probably want to make Instagram + Twitter match for where the seed count is located. I have no attachment to the way we had it before -- do you want to move the TwitterClient seed count stuff around?